docs/windows.md

```diff
@@ -149,14 +149,16 @@ kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/
 
 #### Join Windows worker Nodes
 
-#### 1. (Optional, Test-Only) Install OVS provided by Antrea
+#### 1. (Optional) Install OVS provided by Antrea
```

Comment:
should this be: "1. Install OVS (provided by the Antrea team or your own)"

Reply:
Updated.